framework/filters/Cors.php

```diff
@@ -80,6 +80,10 @@ class Cors extends ActionFilter
      */
     public $actions = [];
     /**
+     * @var bool handle the preflight check and don't proceed with the execution of the request.
+     */
+    public $handleOptions = true;
```

Comment:
I guess if you have a setup where you want code to be executed on an OPTIONS request, you can do this, but you can't use authentication then. I would not use it, but that could be a use case.